plugin.xml

```diff
@@ -31,13 +31,13 @@
 
     <engines>
       <engine name="cordova" version=">=3.1.0" /><!-- Needs cordova/urlutil -->
+      <engine name="cordova-ios" version=">=4.0.0" />
     </engines>
 
     <!-- android -->
     <platform name="android">
         <js-module src="www/inappbrowser.js" name="inappbrowser">
             <clobbers target="cordova.InAppBrowser.open" />
-            <clobbers target="window.open" />
```

Comment:
@GFOutsystems isn't this a breaking change for us? What is the clobber that we're using in the OutSystems wrapper?